assets/src/components/propertiesPanel/minischedule.tsx

```diff
@@ -42,22 +42,53 @@ export interface Props {
 
 export const MinischeduleRun = ({ vehicleOrGhost }: Props): ReactElement => {
   const run: Run | null | undefined = useMinischeduleRun(vehicleOrGhost.tripId!)
+  return (
+    <Minischedule runOrBlock={run} vehicleOrGhost={vehicleOrGhost} view="run" />
```

Comment:
@skyqrose rather than pass a separate and redundant `view` argument, could we instead have a [type predicate](https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-type-predicates) to determine if `runOrBlock` is a run or if it's a block?

Reply:
I tried that, and it did make a bunch of things nicer, but we wouldn't be able to say something different for `No run found`/`No block found`, since we wouldn't have an object to call the type predicate on.

Reply:
Oh right. We could probably do something clever with the type system to avoid the redundancy, but that's going a little overboard.